crates/rooch-rpc-api/src/jsonrpc_types/rooch_types.rs

```diff
@@ -256,7 +256,7 @@ impl AnnotatedCoinStoreView {
         }
     }
 
-    pub fn get_coin_type(&self) -> StructTagView {
+    pub fn get_coin_type_(&self) -> StructTagView {
```

Comment:
Why need `_`?

Reply:
This coin type contains `0x3::coin::Coin<>`,  corresponding to the type of filed_value, in order to distinguish the cointype in BalanceInfo.

https://github.com/rooch-network/rooch/blob/467707e7f294d23277ac8aa7fd026fd0aea7354f/crates/rooch-types/src/framework/coin.rs#L79